src/components/Fullscreen.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {FC, useState, useEffect} from 'react';
+import Box from './Box';
+
+const Fullscreen: FC = ({children}) => {
+	const [size, setSize] = useState({
+		columns: process.stdout.columns,
+		rows: process.stdout.rows
+	});
+
+	useEffect(() => {
+		function onResize() {
+			setSize({
+				columns: process.stdout.columns,
+				rows: process.stdout.rows
+			});
+		}
+
+		process.stdout.on('resize', onResize);
+		process.stdout.write('\u001B[?1049h');
```

Comment:
This should use the `ansi-escapes` package. If these escapes don't exist there, they should be added.